Wikipedia/Code/AbuseFilterAlertView.swift

```diff
@@ -0,0 +1,68 @@
+
+enum AbuseFilterAlertType {
+    case warning
+    case disallow
+}
+
+class AbuseFilterAlertView: UIView, Themeable {
```

Comment:
Quick and dirty conversion of `AbuseFilterAlert.m` - guessing most of this will go away once Carolyn tweaks the old design.